charms/docker/docker.py

```diff
@@ -0,0 +1,87 @@
+import os
+import subprocess
+
+from shlex import split
+
+from .workspace import Workspace
+
+
+class Docker:
+    '''
+    Wrapper class to communicate with the Docker daemon on behalf of
+    a charmer. Provides stateless operations of a running docker daemon
+    '''
+
+    def __init__(self, socket="unix:///var/run/docker.sock", workspace=None):
```

Comment:
OK I see the issue, it is using the third slash as the root dir. If this is correct, you should fix all the docstrings to reflect 3 slashes after unix: